hw/ip/aon_timer/data/aon_timer_testplan.hjson

```diff
@@ -76,7 +77,7 @@
       desc: '''
         Includes possible values of all registers of AON timer. For 32b registers, bins are
         introduced in order to simplfy coverage. Maximum and minimum values are collected in
-        a seperate bin to guarantee that they are checked. 32 separate bins are auto-generated 
+        a seperate bin to guarantee that they are checked. 32 separate bins are auto-generated
```

Comment:
nit: somewhat unrelated - but since we;re touching the file anyways: seperate -> separate

Reply:
Fixed.